iron-overlay-manager.html

```diff
@@ -78,10 +78,58 @@
     },
 
     /**
+     * Brings the overlay at the specified index to the front.
+     * @param {number|Element} i
+     */
+    bringToFront: function(i) {
+      // If not a number, assume i is the overlay.
+      if (typeof i !== 'number') {
+        i = this._overlays.indexOf(i);
+      }
+      var overlay = this._overlays[i];
+      var lastI = this._overlays.length - 1;
+      // If already the top element, return.
+      if (!overlay || i === lastI) {
+        return;
+      }
+      // Update z-index to be on top.
+      var minimumZ = Math.max(this.currentOverlayZ(), this._minimumZ);
+      if (this._getZ(overlay) <= minimumZ) {
+        this._applyOverlayZ(overlay, minimumZ);
+      }
+
+      // Shift other overlays behind the new on top.
+      while (i < lastI) {
+        this._overlays[i] = this._overlays[i + 1];
```

Comment:
It's up to you. FWIW http://jsperf.com/array-prototype-move/24 1 x splice + 1 x push seems to be the same order of magnitude as this verbose alternative, and the cost is so small I doubt anyone will ever notice it in practice. No need to block on this change if you don't want to.